### ISSUE TITLE
Map not loading

### ISSUE THREAD

**Issue by Lioncat2002** [RobLoach/raylib-tiled#2]
Hello now I have been trying out this library
however it seems to not be working?
![image](https://user-images.githubusercontent.com/74904820/147778964-6983575a-6889-4581-be77-e78d46dffed0.png)
The logs do show that the tileset image has been loaded successfully

I decided to dig into the code a little
and it seems that there is a problem with this if statement 
![image](https://user-images.githubusercontent.com/74904820/147781169-d5f80862-e9ce-475a-b869-10c2d8c5b021.png)
and the tile hash id actually returns 0 which is why the if is returning a False?

**Comment by RobLoach** [comment]
Yeah, likely need to change how the tilesets are found across layers 🤔

**Comment by RobLoach** [comment]
Got it to work! 5 years later lol

**Comment by Lioncat2002** [comment]
Awesome!
I don't even remember creating this issue lol, it's been so long xD
Great work!
